docs/_data/menu-sql.yaml

```diff
@@ -80,8 +80,6 @@
       url: sql-ref-datatypes.html
     - text: Null Semantics
       url: sql-ref-null-semantics.html
```

Comment:
The null semantics is also a sub category of the data types section?

Reply:
I scanned through the null semantics file, seems to me it's not a sub category of the data types section.